app/src/main/java/fi/helsinki/opintoni/web/rest/privateapi/portfolio/jobSearch/PrivateJobSearchResource.java

```diff
@@ -58,13 +58,11 @@ public PrivateJobSearchResource(PermissionChecker permissionChecker, JobSearchSe
         return response(jobSearchService.insert(portfolioId, jobSearchDto));
     }
 
-    @RequestMapping(
-        method = RequestMethod.DELETE,
-        value = "/{jobSearchId}")
+    @RequestMapping(method = RequestMethod.DELETE)
     public ResponseEntity<Void> insert(@UserId Long userId,
-                                       @PathVariable Long jobSearchId) {
-        permissionChecker.verifyPermission(userId, jobSearchId, JobSearch.class);
-        jobSearchService.delete(jobSearchId);
+                                       @PathVariable Long portfolioId) {
+        permissionChecker.verifyPermission(userId, portfolioId, Portfolio.class);
```

Comment:
`permissionChecker`in käyttö tässä turhaa (portfolio-tasolla oikeus tarkistetaan jo aiemmin kutsun käsittelyssä `PrivatePortfolioInterceptor`illa)